lib/simplecov.rb

```diff
@@ -54,6 +54,42 @@ def start(profile = nil, &block)
       start_coverage_measurment
     end
 
+    #
+    # Collate a series of SimpleCov result files into a single SimpleCov output.
+    # You can optionally specify configuration with a block:
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"]
+    #    OR
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"], 'rails' # using rails profile
+    #    OR
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"] do
+    #     add_filter 'test'
+    #   end
+    #    OR
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"], 'rails' do
+    #     add_filter 'test'
+    #   end
+    #
+    # Please check out the RDoc for SimpleCov::Configuration to find about available config options
```

Comment:
Might wanna mention README for a more in depth description of what collate does

Reply:
Fixed by https://github.com/colszowka/simplecov/pull/780/commits/99c0495ac8fcb5d0a268dfc4f638b4c086c6c97e.